src/services/LocalStorageService/index.js

```diff
@@ -0,0 +1,19 @@
+class LocalStorageService {
```

Comment:
The name looks too generic. I would expect facade for local storage that implements basic functionality. But `getAppData()` does not have parameters, it's implementation for particular need. 
We an call it `AppDataStorage` or something similar.

Reply:
You are right, renamed to `AppDataStorage`